xbmc/cores/DllLoader/ldt_keeper.c

```diff
@@ -263,7 +263,7 @@ ldt_fs_t* Setup_LDT_Keeper(void)
   Setup_FS_Segment();
 
   ldt_fs->prev_struct = malloc(8);
-  *(void**)array.base_addr = ldt_fs->prev_struct;
+  array.base_addr = (uintptr_t)(ldt_fs->prev_struct);
```

Comment:
static_cast please

Reply:
That's C code.

Reply:
ha should have looked closer. ok from me